openstack/clustering/v1/webhooks/requests.go

```diff
@@ -0,0 +1,43 @@
+package webhooks
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// TriggerOpts Webhooks request parameters
+type TriggerOpts struct {
+	// V (Optional)	query	string	The webhook implementation version requested.
+	V string `q:"V, omitempty"`
+
+	// params (Optional)	query	object	The query string that forms the inputs to use for the targeted action.
+	Params string `q:"params, omitempty"`
```

Comment:
The `q` tag is internal to Gophercloud and doesn't have an `omitempty` modifier. However, I believe `params` is actually in the body of the request as seen here:

https://github.com/openstack/senlin/blob/e5bc16b3fea2f10dead365f983c5a937f23d72b0/senlin/api/openstack/v1/webhooks.py#L29-L30

So I believe this should be:

```go
Params string `json:"params,omitempty"`
```

But still, I don't think `string` is the correct data type here. Looking at the unit tests for webhooks (https://github.com/openstack/senlin/blob/9ce18436035fade18a29e9b47f9cf7389c55da81/senlin/tests/unit/objects/requests/test_webhooks.py), it looks as though `params` is a `map[string]string`.

So perhaps:

```go
Params map[string]string `json:"params,omitempty"`
```

Reply:
Need to look into this one. Doc specified query string.